### PR TITLE
Clarify text

### PR DESCRIPTION
**Pull request by MisinformedDNA**
rbenv-install.sh is not mentioned anywhere else, but it seems pretty clear that it is referring to the commands listed above it.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@MisinformedDNA__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by rmcmurray** [comment]
I agree, it's probably best to reword this as "The commands listed above will perform the following actions"

**Comment by deneha** [comment]
@MisinformedDNA Thank you for your contribution.  We've merged the new content.
